.circleci/config.yml

```diff
@@ -49,7 +49,7 @@ jobs:
 
       - run:
           name: Build the Docker image
-          command: make clean docker-image
+          command: make clean _vendor-download docker-image
```

Comment:
do we have this target?

Reply:
It was missing, I included it in the Makefile